tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -267,6 +267,40 @@ def _run(self):
                                     rebal_files=volume.rebal_files,
                                     rebal_data=volume.rebal_data
                                 ).save()
+
```

Comment:
Are you running this code always even if the volumes are present? In that case there is a possibility of overwriting the status provided by gstatus?